format/spec.md

```diff
@@ -193,10 +193,38 @@ Notes:
 
 For details on how to serialize a schema to JSON, see Appendix C.
 
+#### Default value
+Default values can be assigned to top-level columns or nested fields. Default values are used during schema evolution when adding a new column. The default value is used to read rows belonging to the files that lack the column or nested field prior to the schema evolution.
+
+Currently, when a default value for a column or nested field is set, it is considered an incompatible change to change it to another value. However, changing default values is allowed when calling the `allowIncompatibleChanges` API explicitly. Changing default values is discouraged since the occurrence of a rewrite may determine whether the new or old default value is returned during the read.
+
+Default values are encoded in JSON format. The representation depends on the type of the corresponding field. The mapping of types and their corresponding default value JSON representation is described in the following table.
+
+| type               | json type     | example                                | note                                                                                                                                                                                                                                         |
```

Comment:
This probably would go in the Serialization Json Format appendix down at the bottom

Reply:
I think we should also match the other tables which don't use vertical alignment

Reply:
Regarding the alignment, I think my intellij automatically did this reformatting. But this shouldn't affect the rendered md file view, and it seems it alignment also makes the raw file more human-readable too, so I suggest we can keep this change? WDYT?

Reply:
I find that alignment is helpful for editing, but hard to maintain. If IntelliJ does this automatically, then that's good to know.

We already have a mix, so for a new table this is okay. But we should definitely remove the reformatting of an existing table.